client/app/helper/controller-navigation-helper.service.js

```diff
@@ -11,6 +13,13 @@ class ControllerNavigationHelper {
 
         return url;
     }
+
+    getConstant (constantName) {
+        const path = _.get(this.URLS, constantName);
+
+        return this.OvhApiMe.Lexi().get().$promise
+            .then(me => path[me.ovhSubsidiary]);
```

Comment:
You should add a default behavior, path["FR"] when there is link for this particular subsidiary.

Reply:
Yap, I have thought about it. 
But we should handle target in this case because there's not all constant FR in CA and US.. So keeping this in mind should i fallback to "FR" ?